samples/stock/stocktools/scrollable-popup/demo.js

```diff
@@ -0,0 +1,15 @@
+Highcharts.stockChart('container', {
+    mapNavigation: {
+        enabled: true
+    },
+    stockTools: {
+        gui: {
+            enabled: true,
+            buttons: ['indicators']
+        }
+    },
+    series: [{
+        type: 'line',
+        data: Array.from(Array(50)).map(() => Math.random() * 10)
```

Comment:
isn't it better to generate the random array this way? 
`Array.from({length: 50}, () => Math.random() * 10)`

Reply:
it does look slightly better thanks!